src/gateway/game.gateway.ts

```
@@ -51,51 +50,123 @@ export class GameGateway implements OnGatewayConnection, OnGatewayDisconnect {
   }
 
   /**
-   * Start a new game vs AI
+   * Player joins a multiplayer game as the second player
+   */
+
+@SubscribeMessage('joinMultiplayerGame')
```

ESLint

[error] Insert `··`

(prettier/prettier)